include/fmt/format-inl.h

```diff
@@ -55,7 +55,7 @@ inline fmt::internal::null<> strerror_s(char*, std::size_t, ...) { return {}; }
 FMT_BEGIN_NAMESPACE
 namespace internal {
 
-FMT_FUNC void assert_fail(const char* file, int line, const char* message) {
+FMT_API FMT_FUNC void assert_fail(const char* file, int line, const char* message) {
```

Comment:
I think `FMT_API` is not needed here.

Reply:
Oh, maybe. First, I thought header goes to users, but it seems it is fmt internal stuff?

Reply:
Yep, tested, works without it.

Reply:
It's for the opt-in header-only mode and effectively an internal header.